Package.swift

```diff
@@ -98,6 +102,11 @@ let package = Package(
             dependencies: ["BraintreeCore"],
             path: "Sources/BraintreeSEPADirectDebit"
         ),
+        .target(
+            name: "BraintreeShopperInsights",
+            dependencies: ["BraintreeCore"],
+            path: "Sources/BraintreeShopperInsights"
```

Comment:
I _think_ we get these paths for free - it defaults to `Sources/Target`. I think this can be removed (and probably also removed from the other modules that still have it).

Reply:
Good call, we do! Yeh looks like it can be removed from SEPA. We can do that against `main` in another PR

Reply:
Removed in 39f11666e054f08dacfae280c53400d1578afaca